blocks/library/text/index.js

```diff
@@ -70,12 +67,10 @@ registerBlockType( 'core/text', {
 		const { align, content } = attributes;
 
 		if ( ! align ) {
-			return content;
+			return <p>{ content }</p>;
 		}
 
-		return Children.map( content, ( paragraph ) => (
-			cloneElement( paragraph, { style: { textAlign: align } } )
-		) );
+		return <p style={ { textAlign: align } }>{ content }</p>;
```

Comment:
Outside scope, but given removal of `element` dependency, sooner than later I'd like us to move from inferred `wp.element.createElement` to explicit `import { createElement } from 'element';`. Related: #742